handbook/product/design/design_process.md

```diff
@@ -121,15 +121,57 @@ While asynchronous communication is a core attribute of remote work, key moments
       - Schedule a meeting with engineers to discuss the interaction
   - Tools
       - Figma
-- Metrics collection:
+- **Design to engineering handoff**
+    - Design preperation
+        - Ensure colors utilize the named color styles vs. the color palette where possible 
```

Comment:
I would add some examples:
'Ensure colors utilize the named color styles (ex. web-styles/body-color/light) vs. the color palette (ex. grey-7) where possible

Reply:
Please feel free to add enhancements like this without asking!